api/src/main/java/ai/djl/modality/cv/output/CategoryMask.java

```diff
@@ -183,7 +184,7 @@ private Image getColorOverlay(int[] colors) {
 
     private int[] generateColors(int background, int transparency) {
         int[] colors = new int[classes.size()];
-        colors[0] = background;
+        colors[0] = background & 0xFFFFFF | transparency << 24;
```

Comment:
We assume background contains alpha, this allows to handle background color differently from mask

Reply:
OK